src/cli.rs

```diff
@@ -37,6 +37,9 @@ pub struct Args {
     #[clap(env = "PING_INTERVAL_SEC", hide(true), value_parser=parse_seconds, default_value = "55")]
     pub ping_interval: Duration,
 
+    #[clap(env = "ASSIGNMENT_CHECK_INTERVAL_SEC", hide(true), value_parser=parse_seconds, default_value = "60")]
+    pub assignment_check_interval: Duration,
```

Comment:
It should go under `P2PArgs` because it's not used in HTTP mode

Reply:
done